files/jenkins-slave.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+JENKINS_CONFIG=/etc/sysconfig/jenkins-slave
+
+# Read config
+[ -f "$JENKINS_CONFIG" ] && . "$JENKINS_CONFIG"
+
+if [ -x /sbin/runuser ] ; then
+    RUNUSER=runuser
+else
+    RUNUSER=su
```

Comment:
Aw, good point.  I was thinking of the jenkins master sysconfig which does have the conditional logic.  This seems reasonable then.  I would suggest relocating the `jenkins-slave` script to be under `$libdir` since there is no guarantee that `/usr/local/bin` exists.